tiflash/monitor-tiflash.md

```diff
@@ -8,30 +8,71 @@ aliases: ['/docs/dev/tiflash/monitor-tiflash/','/docs/dev/reference/tiflash/moni
-| `tiflash_raft_read_index_duration_seconds` | The time used by `read_index`. Most time is used for interaction with Leader and retry. |
-| `tiflash_raft_wait_index_duration_seconds` | The time used by `wait_index`, namely the time used to wait until local index >= read_index after the `read_index` request is received. |
+If you use TiUP or TiDB Ansible to deploy the TiDB cluster, the monitoring system (Prometheus & Grafana) is deployed at the same time. For more information, see [Overview of the Monitoring Framework](/tidb-monitoring-framework.md).
+
+The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_exporter, Disk Performance, and so on. A lot of metrics are there to help you diagnose.
+
+TiFlash has three dashboard panels: **TiFlash-Summary**, **TiFlash-Proxy-Summary**, and **TiFlash-Proxy-Details**. The metrics on these panels indicate the current status of TiFlash. The **TiFlash-Proxy-Summary** and **TiFlash-Proxy-Details** panel mainly shows the information of the Raft layer and the metrics are detailed in [Key Monitoring Metrics of TiKV](/grafana-tikv-dashboard.md).
+
+> **Note:**
+>
+> It is recommended that you use TiDB v4.0.5 or later versions for improved monitor on TiFlash.
+
+The following sections introduce the default monitoring information of **TiFlash-Summary**.
+
+## Server
+
+- Store size: The storage size used by each TiFlash instance.
+- Available size: The storage size available for each TiFlash instance.
+- Capacity size: The storage capacity for each TiFlash instance.
+- Uptime: The runtime of TiFlash since last restart.
+- Memory: The memory usage per TiFlash instance.
+- CPU Usage: The CPU utilization per TiFlash instance.
+- FSync OPS: The number of fsync operations per TiFlash instance per second.
+- File Open OPS: The number of `open` operations per TiFlash instance per second.
+- Opened File Count: The number of file descriptors currently opened by each TiFlash instance.
+
+> **Note:**
+>
+> Store size, FSync OPS, File Open OPS, and Opened File Count currently only cover the monitoring information of the TiFlash storage layer and do not cover that in TiFlash-Proxy.
+
+## Coprocessor
+
+- Request QPS: The number of coprocessor requests received by all TiFlash instances. `batch` is the number of batch requests. `batch_cop` is the number of coprocessor requests in the batch requests. `cop` is the number of coprocessor requests that are sent directly via the coprocessor interface. `cop_dag` is the number of dag requests in all coprocessor requests. `super_batch` is the number of requests to enable the Super Batch feature.
+- Executor QPS: The number of each type of dag executors in the requests received by all TiFlash instances. `table_scan` is the table scan executor. `selection` is the selection executor. `aggregation` is the aggregation executor. `top_n` is the `TopN` executor. `limit` is the limit executor.
```

Comment:
As I recall, we used "operator" instead of "executor" in other docs, which conflicts with this metric name, such as [TopN and Limit Operator](https://docs.pingcap.com/tidb/dev/topn-limit-push-down#topn-and-limit-operator-push-down). @TomShawn What do you think?

Reply:
This is actually copied from the existing content in this document. Looks like there's something wrong with the wording...